RealtimeAppBackend/RealtimeApp/views.py

```diff
@@ -42,7 +43,56 @@ def retrieveData(request):
 @csrf_exempt
 def retrieveBluetooth(request):
```

Comment:
remove request argument too, we do not need it. Just leave it as a function and I will use this function later to get real time data